releasenotes/notes/bump-go-to-1.15.13-8745fbc6715b9e4d.yaml

```
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The Agent, Logs Agent and the system-probe are now compiled with Go ``1.15.13``
```

I realize the release notes for the 1.14 and 1.15.11 Go versions were not fully correct either, but let's mention that all agent binaries use 1.15.13:

```suggestion
    All Agent binaries are now compiled with Go ``1.15.13``
```